woocommerce/woocommerce-android
# Shipping Lines Feedback UI

Closes: #11398

### Why
After releasing the new shipping line support feature, we want to collect user feedback.

### Description
This PR adds the `FeedbackDialog` control, which displays the shipping lines' feedback. The PR only handles the UI and animations. This PR also ensures that the logic to display the shipping lines is executed only once after a change in the order's shipping lines. The logic to display the shipping line and the links to the crowdsignal will be added in a different PR.

### Testing instructions
1. Open the app
2. Navigate to the orders tab
3. Tap on add new order (+)
4. Add a product to enable the shipping section
5. Add a new shipping line
6. Check that the feedback control is displayed
7. Tap on close or share feedback to close the control
8. Add/Update/Delete a shipping line
9. Check that the feedback control is not displayed again (we want the control to be displayed only once)

### Images/gif

| Light Mode  | Dark Mode |
| ------------- | ------------- |
| <img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/18119390/a65c7fbe-f9c1-4e8e-858f-5f50a2fc8b73" />  | <img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/18119390/bb8fd458-4105-4bdd-a1e4-4d1ff8cb0bd3" />  |
